backend/src/main/java/fk/prof/backend/model/assignment/AggregationWindowPlanner.java

```diff
@@ -71,6 +80,17 @@ public AggregationWindowPlanner(Vertx vertx,
     this.aggregationWindowDurationInSecs = aggregationWindowDurationInSecs;
     this.policyRefreshBufferInSecs = policyRefreshBufferInSecs;
 
+    processGroupTag = new ProcessGroupTag(processGroup.getAppId(), processGroup.getCluster(), processGroup.getProcName());
```

Comment:
Metrics from fk-prof code are namespaced at process-group level (and in some cases at backend level) but we will publish to cosmos after stripping process group from the metric name and sending that as a tag. This is crucial to avoid metric explosion in cosmos (And us getting blacklisted)